marklogic-data-hub-central/ui/src/pages/Run.test.tsx

```diff
@@ -775,7 +790,7 @@ describe('Verify Add Step function', () => {
         fireEvent.click(step);
 
         // Click to confirm the add in the dialog
-        expect(getByText(`Are you sure you want to add step "${data.steps.data['ingestionSteps'][0].name}" to flow "${data.flows.data[0].name}"?`)).toBeInTheDocument();
+        expect(getByTextWithMarkup(`Are you sure you want to add step ${data.steps.data['ingestionSteps'][0].name} to flow ${data.flows.data[0].name}?`)).toBeInTheDocument();
```

Comment:
Instead of creating a new Matcher function, you can use 'getSubElements', ex:

```
        expect(await(waitForElement(() => getByText((content, node) => {
            return getSubElements(content, node,`Are you sure you to add step ${data.steps.data['ingestionSteps'][0].name} to flow ${data.flows.data[0].name}?`);
        })))).toBeInTheDocument();
```

Please try using that and removing the snippet creating a new function.

Reply:
Nice catch.  I didn't even see the snippet that I could have used underneath.  Thanks for pointing it out.